src/main/java/seedu/address/model/packinglistitem/UniquePackingList.java

```diff
@@ -128,7 +128,7 @@ public void setPackingList(UniquePackingList replacement) {
      *
      * @param items the items
      */
-    public void setPackingList(List<Item> items) {
+    public void setPackingLists(List<Item> items) {
```

Comment:
Same as above, `setPackingList` already means multiple items `setPackingLists` means multiple lists which isn't the intention here


**Suggestions:**
Either (preferred since the above method is `setItem()`)
```suggestion
    public void setItems(List<Item> items) {
```

Or
```suggestion
    public void setPackingList(List<Item> items) {
```